prismatic/util/file_utils.py

```diff
@@ -0,0 +1,16 @@
+import os
+import glob
+
+def get_most_recent_pt_file(directory_path, extension=".pt"):
```

Comment:
consider replacing with this function we use in open_lm with also supports `s3`:

https://github.com/mlfoundations/open_lm/blob/6bb1247d8e668099fdfcacbe6a42d4f3ba87d5cc/open_lm/main.py#L82